scout/server/blueprints/variants/controllers.py

```diff
@@ -1,56 +1,53 @@
 # -*- coding: utf-8 -*-
+import datetime
 import logging
 import os.path
 import urllib.parse
-
+from datetime import date
 from pprint import pprint as pp
 
-from xlsxwriter import Workbook
-
-from datetime import date
-import datetime
+from flask import Response, flash, request, url_for
 from flask_login import current_user
-from flask import url_for, flash, request, Response
 from flask_mail import Message
-from werkzeug.datastructures import MultiDict, Headers
+from werkzeug.datastructures import Headers, MultiDict
+from xlsxwriter import Workbook
 
 from scout.constants import (
-    CLINSIG_MAP,
+    ACMG_COMPLETE_MAP,
     ACMG_MAP,
     ACMG_OPTIONS,
-    ACMG_COMPLETE_MAP,
     CALLERS,
-    SPIDEX_HUMAN,
-    VERBS_MAP,
-    MOSAICISM_OPTIONS,
-    SEVERE_SO_TERMS,
-    MANUAL_RANK_OPTIONS,
     CANCER_TIER_OPTIONS,
+    CLINSIG_MAP,
     DISMISS_VARIANT_OPTIONS,
+    MANUAL_RANK_OPTIONS,
+    MOSAICISM_OPTIONS,
+    SEVERE_SO_TERMS,
+    SPIDEX_HUMAN,
+    VERBS_MAP,
 )
 from scout.constants.acmg import ACMG_CRITERIA
 from scout.constants.variants_export import EXPORT_HEADER, VERIFIED_VARIANTS_HEADER
 from scout.export.variant import export_verified_variants
+from scout.server.blueprints.genes.controllers import gene
```

Comment:
These black changes are a mess to review!

Reply:
Hehe